doc.go

```diff
@@ -187,6 +187,14 @@ last.
 	$ ron target ron:prep
 	$ ron target prep
 
+ron will attempt to find any additional files in the following order:
+
+	1. A -yaml flag filepath config file
+	2. A local ./ron.yaml or in a parent directory
+	3. Any files in the folder ./.ron/*.yaml
```

Comment:
Just a drive by thought while reading this -- if we ever want to put other things in ~/.ron in the future we can put them in subfolders so they don't interfere with this glob loading.